src/ConfigCatClient.php

```diff
@@ -242,11 +270,14 @@ public function getVariationId(string $key, $defaultVariationId, User $user = nu
             }
 
             if (!array_key_exists($key, $settingsResult->settings)) {
-                $this->logger->error("Evaluating getVariationId('" . $key . "') failed. " .
-                    "Value not found for key " . $key . ". " .
-                    "Returning defaultVariationId: " . $defaultVariationId . ". Here are the available keys: " .
-                    implode(", ", array_keys($settingsResult->settings)));
-
+                $this->logger->error("Failed to evaluate setting '{KEY}' (the key was not found in config JSON). " .
+                    "Returning the `{DEFAULT_PARAM_NAME}` parameter that you specified in your application: '{DEFAULT_PARAM_VALUE}'. " .
+                    "Available keys: {AVAILABLE_KEYS}.", [
+                        'event_id' => 1001,
+                        'KEY' => $key,
+                        'DEFAULT_PARAM_NAME' => '$defaultVariationId', 'DEFAULT_PARAM_VALUE' => $defaultVariationId,
+                        'AVAILABLE_KEYS' => implode(", ", array_keys($settingsResult->settings))
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`